py_configuration_files/codejail.py

```diff
@@ -1,3 +1,28 @@
 """Settings for devstack use."""
 
 from codejail_service.settings.local import *  # pylint: disable=wildcard-import
+
+ALLOWED_HOSTS = [
+    # When called from outside of docker's network (dev's terminal)
+    'localhost',
+    # When called from another container (lms, cms)
+    'edx.devstack.codejail',
+]
+
+CODEJAIL_ENABLED = True
+
+CODE_JAIL = {
+    # These values are coordinated with the Dockerfile and AppArmor profile
```

Comment:
Maybe something like this would help people find these references?
```suggestion
    # These values are coordinated with the Dockerfile (edx/public-dockerfiles) and AppArmor profile (codejail.profile)
```